skore/tests/unit/sklearn/comparison/test_report_common.py

```diff
@@ -200,7 +200,7 @@ def test_comparison_report_summarize_pos_label_overwrite(metric):
     result = report.metrics.summarize(scoring=metric).frame().reset_index()
     assert "Label / Average" not in result.columns
     result = result.set_index("Metric")
-    for report_name in report.report_names_:
+    for report_name in list(report.reports_.keys()):
```

Comment:
```suggestion
    for report_name in report.reports_:
```

You can apply this proposition wherever necessary in the test.